packages/components/organisms/f-template-subnav/src/components/TemplateSubNav.vue

```diff
@@ -1,27 +1,102 @@
 <template>
     <div
-        :class="$style['c-templateSubNav']"
         data-test-id="templateSubNav">
-        I am a TemplateSubNav Component (GB)
+        <div
+            :class="$style['c-breadcrumb-container']">
+            <bread-crumbs
+                :links="breadcrumbLinks" />
+        </div>
+
+        <div
+            :class="$style['c-main-container']">
+            <div
+                :class="$style['c-navigation']">
+                <navigation-links
+                    :links="navigationLinks" />
+            </div>
+
+            <div
+                :class="$style['c-account-page']">
+                <slot />
+            </div>
+        </div>
     </div>
 </template>
 
 <script>
+import BreadCrumbs from '@justeat/f-breadcrumbs';
+import NavigationLinks from '@justeat/f-navigation-links';
+import '@justeat/f-breadcrumbs/dist/f-breadcrumbs.css';
+import '@justeat/f-navigation-links/dist/f-navigation-links.css';
+
 export default {
-    components: { },
-    props: { }
+    name: 'TemplateSubNav',
+
+    components: {
+        BreadCrumbs,
+        NavigationLinks
+    },
+
+    props: {
+        breadcrumbLinks: {
+            type: Array,
+            default: () => []
+        },
+        navigationLinks: {
+            type: Array,
+            default: () => []
+        }
+    }
 };
 </script>
 
 <style lang="scss" module>
-.c-templateSubNav {
+
+.c-breadcrumb-container {
+    padding-top: 10px;
+    padding-bottom: 1px;
+}
+
+.c-main-container {
     display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
+    flex-wrap: wrap;
+    gap: 1rem;
+    padding-top: spacing(x5);
+    padding-bottom: spacing(x5);
+}
+
+.c-breadcrumb-container,
+.c-main-container {
     margin: auto;
-    border: 1px solid $color-red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+    max-width: #{$layout-max-width}px;
+    padding-left: #{$layout-margin}px;
+    padding-right: #{$layout-margin}px;
+
+    @include media('<wide') {
+        padding-left: #{$layout-margin--mid}px;
+        padding-right: #{$layout-margin--mid}px;
+    }
+
+    @include media('<narrow') {
+        padding-left: #{$layout-margin--narrow}px;
+        padding-right: #{$layout-margin--narrow}px;
+    }
+}
+
+.c-navigation {
+    flex-basis: 15rem;
+    flex-grow: 1;
+}
+
+.c-navigation,
+.c-breadcrumb-container {
+    @include media('<=mid') {
```

Comment:
I checked that in prod nav is shown from 768px so I think this media query should not include mid
```suggestion
    @include media('<mid') {
```